invoicing_app/tests/tests.py

```diff
@@ -111,29 +122,23 @@ def test_generate_tax_receipt_per_payment_options(self, patch_generate):
         'invoicing_app.management.commands.generate_tax_receipts_old.Command.generate_tax_receipts'
     )
     def test_generate_tax_receipt_event(self, patch_generate):
-        my_user = User.objects.create(
-            username='user_a',
-        )
-        my_event = Event.objects.create(
-            event_name='event1',
-            is_series_parent=False,
-            user=my_user,
-            event_parent=None,
-        )
-        my_pay_opt = PaymentOptions.objects.create(
-            epp_country='AR',
-            accept_eventbrite=False,
+        my_user = UserFactory.build(username='user_1')
+        my_user.save()
+
+        my_event = EventFactory.build(
+            user=my_user
+        )
+        my_event.save()
+
+        my_pay_opt = PaymentOptionsFactory.build(
             event=my_event,
         )
-        my_order = Order.objects.create(
-            status=100,
-            pp_date=str(dt(2020, 3, 8, 0, 0)),
-            changed=str(dt(2020, 3, 10, 0, 0)),
+        my_pay_opt.save()
+
+        my_order = OrderFactory.build(
```

Comment:
nit: take a look to https://factoryboy.readthedocs.io/en/latest/#associations. It may save you code next time